### DISCUSSION TITLE
Modeling different observation data type, administration route and interindividual variability

### DISCUSSION THREAD

**Discussion by nguyen-tranbach** [Open-Systems-Pharmacology/Forum#1839] [Models] [answered]
Dear community,

I have a dataset of NHPs receiving IV bolus and/or SC doses of dolutegravir and radioactive dolutegravir. How can I take into account all this factors in one model, eventually to have only one typical estimate of dolutegravir for all administration route, radioactive/non radioactive molecule and individuals?

Thank you very much for your help!

**Comment by StephanSchaller** [comment]
Dear @nguyen-tranbach,

You can set this up in PK-Sim (depending on the radioactive decay kinetics, and if target-related kinetics (e.g. TMDD) have to be considered).
Add two compounds (radioactive/non radioactive molecule), define a metabolizing process (use a dummy enzyme in all compartments and define a first order decay function) to simulate transformation for radioactive to non.radioactive entity.
Add an administration protocol (advanced) to define both IV and SC administration (SC can be customized injection to skin interstitial space).

Hope this helps.

**Reply by nguyen-tranbach** [reply]
Thank you @StephanSchaller for your reply,

However, I have a problem with the unit used for measurements of radioactive and non radioactive substance (KBq/mL and ng/mL, respectively, the former unavailable in PK-Sim).

For the administration protocole, there are some subjects receiving IV only and others SC only, for which a dose at 0 is not technically tolerated in PK-Sim.

How can these be addressed ?

**Reply by StephanSchaller** [reply, marked as answer]
Yes, it would definitely be needed to get the radioactivity calibrated to concentration measures to make this useful. Your lab should know/have this.
For the simulation of individual profiles, I can recommend using R
